prow/cmd/branchprotector/protect_test.go

```diff
@@ -28,6 +28,7 @@ import (
 	"k8s.io/test-infra/prow/github"
 
 	"github.com/ghodss/yaml"
+	"k8s.io/apimachinery/pkg/util/diff"
```

Comment:
nit: this import block should be above the `test-infra` one

Reply:
Done

Reply:
Done